packages/changelog-extract/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@brisk-docs/changelog-extract",
+  "version": "1.0.0",
+  "description": "This package is used for extracting changelogs from your projects",
```

Comment:
Technically, all we are doing is displaying changelogs. We aren't doing any work to extract them ourselves.